src/liftwing_api/models/liftwing_model.py

```diff
@@ -0,0 +1,12 @@
+import requests
+
+class LiftwingModel:
```

Comment:
This can be an [abstract class ](https://docs.python.org/3/library/abc.html) and the request method can be an abstractmethod that has the following arguments:

- payload : dict
- method :str ="POST"
- headers : dict

Reply:
lets talk about the methods I added